.claude/skills/automation-audit.md

```
@@ -0,0 +1,168 @@
+- Verify commit message follows conventions
+- Verify Co-Authored-By attribution
+
+```bash
+# Check commit details
+gh api repos/jleechanorg/your-project.com/commits/{sha} \
+  --jq '.commit.message, .commit.author.name, .authors'
+```
+
+## Example Output
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)